src/content/smart-bulletins/033.md

```diff
@@ -0,0 +1,48 @@
+
+- Huawei Technologies Company
+- ZTE Corporation
+- Hytera Communications Corp
+- Hangzhou Digital Hikvision Technology Company
+- Dahua Technology Company
+
+Agencies are required to develop policies regarding their use of charge cards in the context of both section 889 prohibitions (Parts A and B). This Smart Bulletin provides operational guidance and information on issues agencies participating in GSA SmartPay program may desire to consider as they develop those policies, including section 889 implications for all business lines (purchase, travel, and fleet cards). 
+
+This Smart Bulletin uses summary language to address complex issues that may develop under certain purchasing circumstances. When such circumstances exist, or are anticipated to arise related to a planned purchase, cardholders and card managers are encouraged to consult with agency acquisition, information technology and/or supply chain risk management policy officials in accordance with agency policy.
+
+This Smart Bulletin does not address section 889-related requirements related to formal contracting procedures. Instead, it focuses on section 889 implications with regard to purchase card micro-purchases, travel and fleet card transactions. Because integrated cards combine these account types, they are not separately addressed.
+
+## Background
+
+GSA SmartPay charge cards are used to obtain over $30 billion in products and services in support of agency missions through approximately 100 million purchase, travel and fleet transactions each fiscal year. Approximately 86% of these transactions are conducted on-line. Most of these transitions are conducted by non-procurement personnel. In FY19, purchase card transactions alone totaled $22 billion, addressing a wide range of products and services. In that same year, approximately $1.2 billion of these purchases (5.7%) occurred in electronics-related Merchant Category Codes (MCCs). This information is meant to provide insight into the relative level of risk and does not capture the full scope of electronic purchases, nor does it capture purchases from merchants whose systems may use equipment/services that are now covered equipment/services. Furthermore, review of charge card purchase data does not indicate any direct purchases from the five specific sources listed above. FY19 travel card spend totaled approximately $9 billion through 44 million transactions with a limited number of airlines and rental car agencies. Hotel spend is much more widespread. There are more than 3 million travel accounts. Fleet card FY19 spend was approximately $2 billion, through almost 31 million transactions, most of which are for fuel for government vehicles. These fueling transactions take place through a large number of independent or franchised service stations.
```

Comment:
GSA SmartPay charge cards are used to obtain over $30 billion in products and services in support of agency missions through approximately 100 million purchase, travel, and fleet transactions each fiscal year. Approximately 86% of these transactions are conducted online. Most of these transitions are conducted by non-procurement personnel. In FY19, purchase card transactions alone totaled $22 billion, addressing a wide range of products and services. In that same year, approximately $1.2 billion of these purchases (5.7%) occurred in electronics-related Merchant Category Codes (MCCs). This information is meant to provide insight into the relative level of risk and does not capture the full scope of electronic purchases, nor does it capture purchases from merchants whose systems may use equipment/services that are now covered equipment/services. Furthermore, a review of charge card purchase data does not indicate any direct purchases from the five specific sources listed above. FY19 travel card spend totaled approximately $9 billion through 44 million transactions with a limited number of airlines and rental car agencies. Hotel spend is much more widespread. There are more than 3 million travel accounts. Fleet card FY19 spend was approximately $2 billion through almost 31 million transactions, most of which are for fuel for government vehicles. These fueling transactions take place through a large number of independent or franchised service stations.